sources/packages/forms/src/form-definitions/institutionprofilecreation.json

```diff
@@ -2509,7 +2484,7 @@
       "labelWidth": "",
       "labelMargin": "",
       "tag": "h2",
-      "className": "category-header-large primary-color",
+      "className": "category-header-large primary-color mt-6",
```

Comment:
Are we expecting to have `mt-6` across many forms, for instance for `category-header-large`?
If yes, it should be a shared class.

Reply:
@andrewsignori-aot Yes for all forms - I am adding a mt-6. But I would not add a mt-6 to the first heading (on forms) and in scenarios like the screenshot below. Just subsequent ones for separation between the sections on forms.

![image](https://user-images.githubusercontent.com/56935198/196821255-bbcb5fe4-1781-42ec-852e-04988d30837d.png)

Reply:
It is a suggestion, you can go ahead with the merge. Just keep in mind that it is a pattern across many forms it would be always good to have a class 😉